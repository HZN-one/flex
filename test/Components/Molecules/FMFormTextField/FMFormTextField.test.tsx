/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMFormTextField } from "@Components";

import { render } from "@testing-library/react";

describe("FMFormTextField", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMFormTextField variant="outlined" testID="input-example" />
    );
    expect(container).toMatchSnapshot();
  });
});
