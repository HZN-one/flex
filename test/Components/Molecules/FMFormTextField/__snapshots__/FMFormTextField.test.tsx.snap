// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMFormTextField should match snapshot 1`] = `
<div>
  <div
    style="display: flex; align-items: center; width: 100%; justify-content: space-between;"
  >
    <div
      class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-1wniyei-MuiTypography-root"
    />
    <div
      class="MuiFormControl-root css-d5nsi3-MuiFormControl-root"
      data-testid="input-example"
    >
      <label
        class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-disabled css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink="false"
        for="component-error"
      >
        
      </label>
      <div
        aria-describedby="component-error-text"
        class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
      >
        <div
          class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
            type="text"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-nnbavb"
            >
              <span
                class="notranslate"
              >
                â€‹
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <p
        class="MuiFormHelperText-root Mui-disabled Mui-error MuiFormHelperText-sizeMedium MuiFormHelperText-contained css-1wc848c-MuiFormHelperText-root"
        id="component-error-text"
      />
    </div>
  </div>
</div>
`;
